docs/tutorials/vqe_with_estimator.ipynb

```diff
@@ -39,7 +39,7 @@
   },
```

Comment:
<p>Have you try running these cells yourself? My experience of this is that it can get confusing if someone is using a virtual environment. I think it's better if we just mention this command in text, not in a jupyter code cell. Like this</p><p> </p><pre class="ql-syntax" spellcheck="false">pip install qiskit-nature
pip install 'qiskit-nature[pyscf]'
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-ibm-runtime/pull/281/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>